src/pages/Info/FAQ.js

```diff
@@ -29,6 +29,10 @@ const questions = [
     "body": "If your college has a large group of people and is within a reasonable distance of Purdue University, we may be able to provide a bus to transport you to Purdue. If transportation is going to be an issue, please contact us at <a href='mailto:team@boilermake.org'>team@boilermake.org</a> and we'll try to figure something out.  We're crossing our fingers Elon comes up with the hyperloop, so we can get a lot more of you here."
   },
   {
+    "title": "Are there travel reimbursements?",
```

Comment:
I figure a large percentage of the emails we've gotten so far have been about travel reimbursements. as long as we make it clear that buses are a possibility I just see this as a way of reducing those mails